datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -339,18 +342,18 @@ def inspect_for_full_name(
         except Exception as e:
             logger.error(f"Column {metadata.name} failed due to {e}")
         values_score = np.round(values_score, 2)
-        debug_info[VALUES] = values_score
+        debug_info.values = values_score
 
     debug_info = compute_name_description_dtype_score(metadata, config, debug_info)
 
     try:
         if (
-            debug_info.get(NAME, None)
-            and int(debug_info[NAME]) == 1  # type: ignore
-            and VALUES in debug_info.keys()
-            and 0.5 > cast(float, debug_info[VALUES]) > 0.1
+            hasattr(debug_info, "name")
```

Comment:
I don't think this hasattr check is necessary anymore, how that we have the dataclass?

Reply:
Removed the hasattr check, and also modified the condition to apply check on prediction_factors_weights which adds more clarity to the code.